pkg/controllers/provisioning/v1alpha1/allocation/suite_test.go

```diff
@@ -87,15 +88,8 @@ var _ = Describe("Allocation", func() {
 
 	Context("Reconcilation", func() {
 		It("should provision nodes for unconstrained pods", func() {
-			pods := []*v1.Pod{
-				test.PodWith(test.PodOptions{
-					Conditions: []v1.PodCondition{{Type: v1.PodScheduled, Reason: v1.PodReasonUnschedulable, Status: v1.ConditionFalse}},
-				}),
-				test.PodWith(test.PodOptions{
-					Conditions: []v1.PodCondition{{Type: v1.PodScheduled, Reason: v1.PodReasonUnschedulable, Status: v1.ConditionFalse}},
-				}),
-			}
-			for _, pod := range pods {
+			ps := []*v1.Pod{pods.Pending(), pods.Pending()}
```

Comment:
This is the exact reason why it's sketchy to name a package with a such a likely variable name (i.e. "pods").

Reply:
We do run into this a lot anyway, eg `packing`, but I'll change back

Reply:
Yeah the packing stuff definitely needs to be resolved.